From 5268047a53bc0e47753a73525e851bb84d3b661e Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Mon, 15 Jan 2024 10:22:46 +0100
Subject: [PATCH 0878/1141] arm64: dts: st: remove backlight on stm32mp257f-dk
 board

The backlight must be link to the panel not to the bridge.

Change-Id: Ideedc1dae9dc0376a14ba332871c9aa5b2a9dd35
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/351202
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
index 527544df573e..6ee43bac12b0 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -453,7 +453,6 @@ &lvds {
 	status = "okay";
 	vdd-supply =  <&scmi_vddcore>;
 	vdda18-supply = <&scmi_v1v8>;
-	backlight = <&panel_lvds_backlight>;
 	ports {
 		#address-cells = <1>;
 		#size-cells = <0>;
-- 
2.39.2

